Jamulus.pro

```diff
@@ -237,7 +237,7 @@ win32 {
 } else:android {
     ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
     ANDROID_VERSION_NAME = $$VERSION
-    ANDROID_VERSION_CODE = $$system(git log --oneline | wc -l)
+    ANDROID_VERSION_CODE = $$str_size($$system(git log --oneline))
```

Comment:
(Ideally we'd not use `git` -- the tarball source should build off line properly.  That's a separate problem...)

Reply:
https://doc.qt.io/qt-6/qmake-function-reference.html#str-size

Says

> Returns the number of characters in the argument.

Is that really equivalent with `wc -l` which gives back the number of lines

Reply:
I think this is https://stackoverflow.com/questions/9629125/versioncode-vs-versionname-in-android-manifest ?
Why don't we just use something like the version number without dots?

Reply:
OK, so it's not important except on releases pushed to Google.